src/Tasks/Microsoft.NET.Build.Extensions.Tasks.UnitTests/Microsoft.NET.Build.Extensions.Tasks.UnitTests.csproj

```diff
@@ -12,7 +12,7 @@
     <OutDirName>Tests\$(MSBuildProjectName)</OutDirName>
   </PropertyGroup>
 
-  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+  <Import Project="Sdk.props" Sdk="RoslynTools.RepoToolset" />
```

Comment:
Personally I really don't like that all projects have to be updated to use a different Sdk.  Also, Sdks don't compose if done this way- there's no way to use both RepoToolset and Microsoft.NET.Sdk.Web for example.

Would it be possible instead to:

- Not have the RepoToolset SDK import Microsoft.NET.Sdk
- Have projects that use RepoToolset do so via Sdk imports in Directory.Build.props and Directory.Build.targets
- Leave the Sdk imports in project files as they were (ie Microsoft.NET.Sdk or Microsoft.NET.Sdk.Web)

We don't need to change this before we update the sdk repo to the new repotoolset, but I would like to consider a change like this for the future.

Reply:
This is the current design. We can discuss changes in future.